specification/operationalinsights/resource-manager/Microsoft.OperationalInsights/preview/2019-08-01-preview/Clusters.json

```diff
@@ -370,9 +361,13 @@
             "modelAsString": true
           }
         },
-        "encryptionKeyUri": {
-          "type": "string",
-          "description": "The Key Vault key or certificate path associated with the Log Analytics cluster."
+        "keyVaultProperties": {
+          "$ref": "#/definitions/keyVaultProperties",
+          "description": "The associated key properties."
+        },
+        "sku": {
```

Comment:
Why is sku being modeled under properties instead of using the top-level sku property (see https://github.com/Azure/azure-resource-manager-rpc/blob/master/v1.0/resource-api-reference.md#put-resource)?

Reply:
Wasn't familiar with this. Thanks!